src/test/scala/chiselTests/ProbeSpec.scala

```diff
@@ -0,0 +1,349 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests
+
+import chisel3._
+import chisel3.probe._
+import circt.stage.ChiselStage
+
+class ProbeSpec extends ChiselFlatSpec with Utils {
+  // Strip SourceInfos and split into lines
+  private def processChirrtl(chirrtl: String): Array[String] =
+    chirrtl.split('\n').map(line => line.takeWhile(_ != '@').trim())
+
+  "Simple probe usage" should "work" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(
+      new RawModule {
+        val a = IO(Output(RWProbe(Bool())))
+
+        val w = WireInit(Bool(), false.B)
+        val w_probe = RWProbeValue(w)
+        define(a, w_probe)
+      },
+      Array("--full-stacktrace")
+    )
+    processChirrtl(chirrtl) should contain("define a = rwprobe(w)")
```

Comment:
This is nice and terse! Interestingly, I've never thought to use an anonymous `new RawModule` like that. Clever.